### DISCUSSION TITLE
Is it possible to do multi select with the Select component?

### DISCUSSION THREAD

**Discussion by devhims** [chakra-ui/chakra-ui#3382] [Q&A]
I couldn't find any prop or anything in the documentation regarding multi-select with the Select component. Can anyone shed some light on it?

**Comment by TheAsda** [comment]
Here is the issue with recommended solution: https://github.com/chakra-ui/chakra-ui/issues/1138

**Reply by devhims** [reply]
Thanks @TheAsda! I got to know about `react-select` which is another great option for multi-select.

**Reply by thedarshanjadhav** [reply]
I tried but in option scroll bar is disable how to enable that

**Comment by csandman** [comment]
There is no official support for a multi select component yet, although they have said they are planning on adding one at some point. If you need an alternative in the meantime, I'd recommend checking out my wrapper for `react-select`.

https://github.com/csandman/chakra-react-select

![image](https://user-images.githubusercontent.com/9214195/149230881-f775030b-89bc-4866-a1db-9aaba1ce5147.png)
